exercises/practice/isbn-verifier/.docs/instructions.md

```diff
@@ -37,6 +43,12 @@ Now, it's even trickier since the check digit of an ISBN-10 may be 'X' (represen
 
 ## Bonus tasks
 
-* Generate a valid ISBN-13 from the input ISBN-10 (and maybe verify it again with a derived verifier).
+If you attempt these tasks, write your own tests for them and include those in your solution too!
```

Comment:
I'm not sure this will work with the web editor. Perhaps we can specify they should be in a separate file or in the main solution?

Reply:
Works in the web editor for me. Try adding `@test false` at the end of an exercise and then running the tests in the web editor.

Reply:
We include the student's solution file within a custom testset so it will all work. It'll even work if the student adds `using Test` because they think they need it or a linter tells them to.

Reply:
> Works in the web editor for me. Try adding @test false at the end of an exercise and then running the tests in the web editor.

Ah yea, that works. I was thinking about adding it to the test file itself.

Reply:
```suggestion
If you attempt these tasks please write your own tests for them in your solution file!
```

Clearer?

Reply:
Yes, definitely!

Reply:
Thanks :)